src/components/carousel/carousel.tsx

```diff
@@ -37,16 +21,36 @@ interface Props {
     lastTitle: string;
     lastSlideAction?: () => void | any;
   };
-  content: WhatsNewContent;
-  locale: string;
+  carouselContent: any[];
+  renderItem: (item: any) => React.JSX.Element;
   theme: Theme;
+  hideButtons?: boolean;
+  moveNext?: number;
```

Comment:
From what I understand of your code, this should be a boolean, I only see 0 and 1 as potential values. Though passing a flag to signal you should move to the next index is quite convoluted. A better way would be to create a reference of the carousel on the parent, and allow the parent to call sth like `carouselRef.moveToNext();`

Reply:
changed trigger type to boolean, left a note in clickup for later consideration.